examples/with-next/app/layout.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Metadata } from 'next'
+import { Inter } from 'next/font/google'
+import { MSWProvider } from './msw-provider'
+
+if (process.env.NEXT_RUNTIME === 'nodejs') {
+  const { server } = require('../mocks/node')
```

Comment:
Would this mean the mocks are bundled with the production code?
Does that have any undesirable side-effects?

Reply:
@votemike, this likely needs tweaking by adding a new env variable like `MOCKS_ENABLED` and checking that one too.